core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -31,14 +31,19 @@ oppia.animation('.oppia-collection-animate-slide', function() {
 });
 
 oppia.controller('CollectionPlayer', [
-  '$scope', 'ReadOnlyCollectionBackendApiService', 'CollectionObjectFactory',
+  '$scope', '$http', 'ReadOnlyCollectionBackendApiService', 'CollectionObjectFactory',
   'CollectionPlaythroughObjectFactory', 'alertsService',
-  function($scope, ReadOnlyCollectionBackendApiService, CollectionObjectFactory,
+  function($scope, $http, ReadOnlyCollectionBackendApiService, CollectionObjectFactory,
     CollectionPlaythroughObjectFactory, alertsService) {
     $scope.collection = null;
     $scope.collectionPlaythrough = null;
     $scope.collectionId = GLOBALS.collectionId;
     $scope.showingAllExplorations = !GLOBALS.isLoggedIn;
+    $scope.showPreviewCard = true;
```

Comment:
Suggest `$scope.previewCardIsShown` -- current naming makes this look like a function since it starts with a verb.